photos/api/serializers.py

```diff
@@ -0,0 +1,11 @@
+from rest_framework import serializers
+
+from photos.models import Photo
+
+
+class PhotoSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Photo
+        fields = ('pk', 'bucket_name', 'file_name', 'caption', 'pub_date')
+        
```

Comment:
We could take out these extra lines at the end. My linter is yelling about them.

Reply:
Removed 👍